device/bgp/bgpsrtev6remoteendpointsubtlv.yaml

```diff
@@ -0,0 +1,34 @@
+components:
+  schemas:
+    BgpSrte.RemoteEndpointSubTlv:
+      description: >-
+        Configuration for BGP remote endpoint sub TLV settings.
+        Question: Do we need to keep? Draft says 
```

Comment:
Anything that is not must is probably better removed. Add it only if someone asks for negative use-case testability.